packages/commons/src/fhir/transformers/index.ts

```diff
@@ -3470,127 +3399,12 @@ export async function buildFHIRBundle(
     entry: [createCompositionTemplate(ref, context)]
   }
 
-  const newFHIRBundle = await transformObj(
+  const newFHIRBundle = transformObj(
     reg as Record<string, unknown>,
     initialFHIRBundle,
     builders,
     context
   )
 
-  const composition = getComposition(newFHIRBundle)
-
-  let isADuplicate = false
-  if (eventType === EVENT_TYPE.BIRTH) {
-    isADuplicate = await hasBirthDuplicates(
-      authHeader,
-      reg as GQLBirthRegistrationInput
-    )
-  } else if (eventType === EVENT_TYPE.DEATH) {
-    isADuplicate = await hasDeathDuplicates(
-      authHeader,
-      reg as GQLDeathRegistrationInput
-    )
-  }
-
-  if (isADuplicate) {
-    composition.extension = composition.extension || []
-    composition.extension.push({
-      url: `${OPENCRVS_SPECIFICATION_URL}duplicate`,
-      valueBoolean: true
-    })
-  }
```

Comment:
This block of code was moved to to `createEventRegistration` in gateway's root resolvers